### ISSUE TITLE
[HOLD for payment 2023-09-27] [HOLD App#21664] [$500] Update the request description to be handled similarly as task description

### ISSUE THREAD

**Issue by mountiny** [Expensify/App#25494]
### Problem

When the description of a request is long, we only show one line in the details and in the edit input and we truncate what does not fit 
![image](https://github.com/Expensify/App/assets/36083550/4ed5e5c5-8cb9-497c-a5cc-78424e993b91)

Thats not ideal

### Solution

Lets mimic task description field for this one in both the moneyRequestView and in the edit form
![image](https://github.com/Expensify/App/assets/36083550/290811ca-01fd-4edd-aa3c-ef28052897fb)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0160a97b98c4b48f40</li>
        <li>Upwork Job ID: 1692628331709181952</li>
        <li>Last Price Increase: 2023-08-18</li>
<li>Automatic offers: </li>
<ul>
<li>daordonez11 | Contributor | 26204376</li>
</ul></li>
    </ul>
</details>

**Comment by mountiny** [comment]
Might handle it on my own

**Comment by daordonez11** [comment]
Proposal based on desired experience by @mountiny 
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
We should unify the experience in both Task and IOU

### What is the root cause of that problem?
Even though both pages use practically the same components styles are different
Step 1 Report:
In `MoneyRequestView`include ` wrapperStyle={[styles.pv2, styles.taskDescriptionMenuItem]}` in the description element copying the styles of `TaskView`

Step 2 Edit Desc:
In `EditRequestDescriptionPage` include this in the props of the TextInput emulating the style of `TaskDescriptionPage`
```js
autoGrowHeight
containerStyles={[styles.autoGrowHeightMultilineInput]}
textAlignVertical="top"
```

Solution working:



### What changes do you think we should make in order to solve the problem?


### What alternative solutions did you explore? (Optional)

**Comment by JmillsExpensify** [comment]
Taking this as part of wave3

**Comment by mountiny** [comment]
I will make this external to focus on other issues myself

Proposal from @daordonez11 look sgood to me, thank you! Can you raise a PR please?

**Comment by mountiny** [comment]
Making this $500 as its quite simple change

**Comment by ishpaul777** [comment]
I would love to take this

**Comment by ishpaul777** [comment]
just realized issue is assigned already. pleasa ignore above comment.

**Comment by mountiny** [comment]
No worries, there are many more with the `Help Wanted` label :bow:

**Comment by nekores** [comment]
Hi there, 
I have sent a job application to your job post on Upwork.
I will fix this issue ASAP, It's a CSS issue, a very easy task. I will fix this immediately.
Here is my Upwork: https://www.upwork.com/freelancers/nekores
Here is my CV: https://sajidullah.com/

I am waiting for your response.

Regards
Sajid U.

**Comment by nekores** [comment]
Contributor details
Your Expensify account email: sajidxullah@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/nekores

**Comment by daordonez11** [comment]
Actually, that [PR](https://github.com/Expensify/App/pull/21664/files) points to different files than the ones from this proposal @aimane-chnaif . Or is it another PR?

**Comment by burczu** [comment]
@aimane-chnaif I'm confused a little bit - should we proceed with the PR from @daordonez11?

**Comment by mountiny** [comment]
@aimane-chnaif isnt that previous PR only for creation but not for money request edit?

**Comment by mountiny** [comment]
@aimane-chnaif Ok I see, I think the Pr you have linked has already long history, lets merge it with the current state and this can be a fast follow up for the edit money request.

**Comment by mountiny** [comment]
Sounds good to me, lets just divide this up as it seems like the previous PR has been around for a while so better to not make it bigger

**Comment by mountiny** [comment]
@aimane-chnaif let me know if there is anything I could do to help with the other PR

**Comment by daordonez11** [comment]
Created [slack thread](https://expensify.slack.com/archives/C01GTK53T8Q/p1692723237220349) to discusss max limit of characters. Also as per https://github.com/Expensify/App/issues/25494#issuecomment-1686306527 I will not work on handling multiline on header, @aimane-chnaif let me know if you guys require help on handling that header.

**Comment by mountiny** [comment]
@aimane-chnaif I think we should put this on hold for https://github.com/Expensify/App/pull/21664

**Comment by mountiny** [comment]
Soon to be merged

**Comment by burczu** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

* [ ]  [@burczu] The PR that introduced the bug has been identified. Link to the PR: I think it worked this way from the beginning.
* [ ]  [@burczu] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: n/a
* [ ]  [@burczu] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: n/a
* [ ]  [@burczu] Determine if we should create a regression test for this bug.
* [ ]  [@burczu] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
* [ ]  [@JmillsExpensify] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by mountiny** [comment]
@JmillsExpensify bump on this one please. Its $500 to @daordonez11 only.

**Comment by JmillsExpensify** [comment]
Payment summary: $500 payment to @daordonez11. Just sent an offer as well!

**Comment by JmillsExpensify** [comment]
All paid out!
